app/models/webhook.rb

```diff
@@ -33,11 +33,11 @@ class Webhook < ApplicationRecord
   validates :secret, presence: true, length: { minimum: 12 }
   validates :events, presence: true
 
-  validate :validate_events
+  validate :events_validation_error, if: :invalid_events?
```

Comment:
Why this change?

Reply:
This change is not relevant to or required by the `normalizes` introduction. Made this change as a small refactor while adding the spec coverage here prior to adding the `normalizes`. Makes it easier to see in coverage report that both parts were actually run. Don't feel strongly about this if you hate it.